k8-operator/controllers/infisicalsecret_controller.go

```diff
@@ -69,6 +75,31 @@ func (r *InfisicalSecretReconciler) handleFinalizer(ctx context.Context, infisic
 	return nil
 }
 
+func (r *InfisicalSecretReconciler) handleManagedSecretDeletion(a client.Object) []ctrl.Request {
+	var requests []ctrl.Request
+	infisicalSecrets := &secretsv1alpha1.InfisicalSecretList{}
+	err := r.List(context.Background(), infisicalSecrets)
+	if err != nil {
+		fmt.Printf("unable to list Infisical Secrets from cluster because [err=%v]", err)
+		return requests
+	}
+
+	for _, infisicalSecret := range infisicalSecrets.Items {
```

Comment:
Just so i'm on the same page. When the delete event is captured, we add details of where the managed secret was deleted so that our secret reconciler gets triggered for that name/namespace?

Reply:
Yes, that's correct. When a delete event is captured, we add the details of where the managed secret was deleted. This ensures that our secret reconciler gets triggered for the corresponding name and namespace.